cloudproxy-ui/src/main.js

```diff
@@ -0,0 +1,13 @@
+import '@babel/polyfill'
+import 'mutationobserver-shim'
+import Vue from 'vue'
+import './plugins/bootstrap-vue'
+import App from './App.vue'
+import { ListGroupPlugin } from 'bootstrap-vue'
+Vue.use(ListGroupPlugin)
+
+Vue.config.productionTip = false
+
+new Vue({
+  render: h => h(App),
+}).$mount('#app')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/claffin/cloudproxy/pullRequest?prid=7329682)
```suggestion
}).$mount("#app")
```